packages/svelte/src/lib/useForm.ts

```diff
@@ -0,0 +1,278 @@
+import type {
+  ActiveVisit,
+  Errors,
+  Method,
+  Page,
+  PendingVisit,
+  Progress,
+  RequestPayload,
+  VisitOptions,
+} from '@inertiajs/core'
+import { router } from '@inertiajs/core'
+import type { AxiosProgressEvent } from 'axios'
+import cloneDeep from 'lodash/cloneDeep'
+import isEqual from 'lodash/isEqual'
+import { writable, type Writable } from 'svelte/store'
+
+interface InertiaFormProps<TForm extends Record<string, unknown>> {
```

Comment:
We can improve a little the typing here using the same types that React uses.
See here https://github.com/jamesst20/inertia/blob/main/packages/svelte5/src/lib/useForm.svelte.ts#L19

I also was able to get rid of all ts-ignore